walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -20,47 +20,25 @@
 
 public final class FindMeetingQuery {
 
-  // Add a time range to Collection results
-  private void addRangeToResults(int validRangeStart, int validRangeEnd,
-      boolean inclusive, Collection<TimeRange> results, long duration) {
+  // Add a time range to validRanges, given the time range
+  // start and end times and whether the time range should include
+  // the endpoints, if it is at least as long as duration.
+  // Return true time range is added, false otherwise.
+  private boolean addRange(int validRangeStart, int validRangeEnd,
```

Comment:
Why not just pass a TimeRange object with the start, end, and inclusive parameters?